Initial update for ratification of extensions EXT_mesh_gpu_instancing…

DrX3D opened the pull request:
Minor changes to **Status** to indicate that these two extensions are ready to start the ratification process.

- EXT_meshopt_compression
- EXT_mesh_gpu_instancing

Needs approval from Geospatial DTSG and 3D Formats before ratification starts

javagl commented:
Assuming that this PR is intended to capture further updates or modifications during the ratification process: The `samples` folder should be removed, because of the issue at https://github.com/KhronosGroup/glTF/issues/1855

DrX3D commented:
Added EXT_texture_webp to the list of extensions changed. In all cases only the status was changed to include 'Release Candidate'.

lexaknyazev commented:
Are you sure that "Release Candidate" is the right status here? It's usually used for marking _new_ extensions as ready for external implementors.

These could remain "Complete" until the ratification ends.

DrX3D commented:
Changed status on all 3 extensions to 'Complete, Release Candidate for Khronos Ratification' and removed 'samples' folder as per above comments. @emackey recommends using 'Release Candidate' for any extension entering Khronos ratification.
